pyproject.toml

```diff
@@ -1,7 +1,56 @@
+[tool.poetry]
+name = "langchain"
+version = "0.0.26"
+description = "Building applications with LLMs through composability"
+authors = []
+license = "MIT"
+readme = "README.md"
+repository = "https://www.github.com/hwchase17/langchain"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+pydantic = "^1.10.2"
+SQLAlchemy = "^1.4.44"
+requests = "^2.28.1"
+PyYAML = "^6.0"
+numpy = "^1.23.5"
+
+[tool.poetry.group.test.dependencies]
```

Comment:
@hwchase17 any opinion on where these dependencies should be grouped? Generic `dev` group?

* `playwright`
* `manifest-ml`
* `jupyter`

I'm not clear on how they're being used today.

Reply:
manifest ml should go in  llms (the only reason it wasnt before was the 3.8 issue)

for the other two, yeah generic dev group sounds perfect